df.viewscreen.xml

```diff
@@ -1144,6 +1144,7 @@
             <enum-item name='Crimes'/>
             <enum-item name='Suspects'/>
             <enum-item name='ConvictChoices'/>
+            <enum-item name='InterrogateChoices' since='v0.47.01'/>
```

Comment:
I'm late to the party, but this looks perfect! Enum changes are pretty much never OS-specific, and you can usually get by with just the xmllint check letting you know if anything broke for those (but thanks for testing!).